src/@types/aquarius/BaseQueryParams.d.ts

```diff
@@ -11,6 +11,7 @@ interface BaseQueryParams {
   sortOptions?: SortOptions
   aggs?: any
   filters?: FilterTerm[]
+  andOrFilters?: FilterTerm[][]
```

Comment:
can we think of a better name for what this is used for? andOr is still relative to something

Reply:
Better call it staticFilters. Fixed!